src/style.css

```diff
@@ -1 +1,141 @@
-/* add styles here */
+
+/* @font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+
+} */
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+/* html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 18px;
+  line-height: normal;
+} */
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+body {
+  /* font-family: Avenir, sans-serif; */
+  padding: 0;
+}
+
+.search-panel {
+  margin-top: 20px;
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.search-bar {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  color: hsl(212, 23%, 31%);
+  line-height: normal;
+  font-style: normal;
+  font-weight: 300;
+  height: 70px;
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
It's better practice to put the box-sizing to this block `*{}` as it this be applied in everywhere